### PR TITLE
Replace deprecated `HashState` with `BuildHasher`.

### LINKED ISSUE #20: Stop using a wildcard for `linked-hash-map` dependency

### PR DESCRIPTION
**Pull request by zonyitoo**
Also fixed #20 . Depend on linked-hash-map of v0.0.8

### CODE REVIEW

**Comment by FlashCat** [comment]
r? @Gankro

(I've picked a reviewer for you, use r? to override)

File: Cargo.toml [hunk 1]
```diff
@@ -15,4 +15,4 @@ keywords = ["data-structures"]
 readme = "README.md"
 
 [dependencies]
-linked-hash-map = "*"
+linked-hash-map = "^0.0.8"
```
Comment (apasel422) [inline Cargo.toml]: Is there a difference between this and

``` toml
linked-hash-map = "0.0.8"
```

Comment (zonyitoo) [inline Cargo.toml]: You are right, there is no different between them ... Fixing it

**Comment by apasel422** [comment]
Thanks! I'll push a new version of this to crates.io later today.

**Comment by zonyitoo** [comment]
Thank you very much.

**Comment by apasel422** [comment]
This was published to crates.io as `0.0.6`.

**Pull request merged**